grails-app/i18n/messages_nl.properties

```diff
@@ -482,7 +482,7 @@ default.notes.label=Notities
 default.null.message=Eigenschap [{0}] van klasse [{1}] kan niet null zijn
 #default.number.format                 = 0
 default.numItems.label=\# items
-default.number.format=\#\#\#.\#\#\#.\#\#0,00
+default.number.format=###,###,##0.00
```

Comment:
I think all the `#` require backslash escape, because this file supports comments starting with `#`

Reply:
I would have guessed you were right, but check messages.properties and all of the other .properties files.

Reply:
:thinking:

Reply:
Yup, looks like you're right, then you can disregard this comment, and it lgtm